### ISSUE TITLE
View.propTypes is deprecated

### ISSUE THREAD

**Issue by davidroman0O** [jineshshah36/react-native-nav#19]
Hi,

All the files should replace View.propTypes because it's deprecated.
https://github.com/react-community/react-navigation/issues/1352

Replace it by ViewPropTypes.

Thx!

**Comment by jineshshah36** [comment]
I just updated the master to use the prop-types module. Will this solve the deprecation problem? If so, please let me know and I will release this a patch release on npm

**Comment by nhducit** [comment]
@jineshshah36 

```
import {Text, View, ViewPropTypes} from 'react-native';
```
instead of:
```
export default class SomeClass extends React.PureComponent {
  static propTypes = {
    style: View.propTypes.style,
  }
  // ...
}
```

https://github.com/facebook/react-native/issues/13999

**Comment by jineshshah36** [comment]
Yes, that has been updated. Please send a pull request with changes if I did it wrong.

**Comment by matt-oakes** [comment]
@jineshshah36 It looks like you fixed it to me! Can you release the change to npm so other modules which depend on this can fix the warning too, please?

**Comment by jineshshah36** [comment]
yes, perfect

**Comment by jineshshah36** [comment]
released as v2.0.1
